tests/functional/analytics/test_google_analytics.py

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.analytics.index import AnalyticsTestPage
+
+
+def assert_ga_loaded(selenium):
+    # if the datalayer list doesnt contain the gtm.load event then GA has not been loaded or it has been blocked
+    data_layer = selenium.execute_script("return window.dataLayer")
+    assert any("event" in layer and layer["event"] == "gtm.load" for layer in data_layer), (
```

Comment:
Suggestion (non-blocking): Seeing as we're testing third party code here, I do wonder if we'll see some intermittent failures if GA is slow to load. Is it possible to add a `wait.until` condition here, before we `assert`? This might be a good safeguard against network flakiness.

Reply:
Good call out, ty